autotest/test_util_2d_and_3d.py

```diff
@@ -614,3 +614,40 @@ def test_mflist_fromfile(function_tmpdir):
     flx_array = wel.stress_period_data.array["flux"][0]
     for k, i, j, flx in zip(wel_data.k, wel_data.i, wel_data.j, wel_data.flux):
         assert flx_array[k, i, j] == flx
+
+
+def test_pathlib_support():
```

Comment:
Use the `tmp_path` fixture:
```suggestion
def test_pathlib_support(tmp_path):
```
then follow guidelines [documented here](https://docs.pytest.org/en/latest/how-to/tmp_path.html).